functions/ldap-config.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+if ( ENVIRONMENT === 'production' ) {
+	$amu_ldap_config = array(
```

Comment:
This information probably shouldn't be in our code on live. Can you look at the possibility of moving this to a plugin option? Apologies as I know this is more work but as we have this as open source, it needs to be configurable. Thanks Richard.

Reply:
We use this in a couple of plugins and is really just an environmental var set in the config. Not too sure it would make sense to move to a site option, but probably worth discussing

Reply:
After discussion with @richeduni, we have agreed to add this in as a network option so that the LDAP config settings can be set on the front end rather than through the code. I will leave this issue open to track this development.

Reply:
this has been moved to options now